widgetssdk/src/main/java/com/glia/widgets/view/dialog/base/DialogViewFactory.kt

```diff
@@ -16,13 +16,13 @@ import com.glia.widgets.view.dialog.screensharing.ScreenSharingDialogViewInflate
 import com.glia.widgets.view.dialog.screensharing.VerticalScreenSharingDialogViewInflater
 import com.glia.widgets.view.dialog.update.UpgradeDialogViewInflater
 import com.glia.widgets.view.dialog.update.VerticalUpgradeDialogViewInflater
-import com.glia.widgets.view.unifiedui.deepMerge
+import com.glia.widgets.view.unifiedui.nullSafeMerge
 import com.glia.widgets.view.unifiedui.theme.AlertDialogConfiguration
 import com.glia.widgets.view.unifiedui.theme.UnifiedTheme
 
 internal class DialogViewFactory(context: Context, uiTheme: UiTheme, unifiedTheme: UnifiedTheme?) {
     private val configuration: AlertDialogConfiguration = uiTheme.alertTheme(context).run {
-        copy(theme = this.theme.deepMerge(unifiedTheme?.alertTheme)!!)
+        copy(theme = theme nullSafeMerge unifiedTheme?.alertTheme)
```

Comment:
Could you please explain, for my understanding, why do you use `nullSafeMerge` instead of `merge` here?

Reply:
because of the return type, `nullSafeMerge` returns non-null value.